interfaces/builtin/iio.go

```diff
@@ -113,7 +111,17 @@ func (iface *iioInterface) AppArmorConnectedPlug(spec *apparmor.Specification, p
 	deviceName := strings.TrimPrefix(path, "/dev/")
 	snippet = strings.Replace(snippet, "###IIO_DEVICE_NAME###", deviceName, -1)
 
+	// Add a snippet for various device specific rules, except for sysfs write
+	// access that are specialized below.
 	spec.AddSnippet(snippet)
+	// This part contains two sets of "**" that can use exponential memory when
+	// compiling without -O no-expr-simplify, so handle it with the parametric
+	// snippet workaround. Only one instance of this line will show up in the
+	// resulting profile.
+	spec.AddParametricSnippet("/sys/devices/**/###PARAM###/** rwk,", deviceName)
```

Comment:
This expands to:
```
/sys/devices/**/{iio:device1,iio:device3}/** rwk,
```

It would be nicer if this expanded to:
```
/sys/devices/**/iio:device{1,3}/** rwk,
```

Since we know that iioControlDeviceNodePattern ensures that deviceName must be of the form iio:deviceN, could we adjust this to be:
```
spec.AddParametricSnippet("/sys/devices/**/iio:device###PARAM###/** rwk,", deviceName[10:]) # len("iio:device") is 10
```

or similar? If changing, please adjust my comment suggestion above accordingly.

Reply:
Indeed. Adjusting now.

Reply:
Done